src/post/post.service.ts

```diff
@@ -492,6 +504,174 @@ export class PostService {
         };
     }
 
+    /**
+     * Get posts for an account using their handle
+     *
+     * @param handle - The handle to look up (e.g., "@username@domain")
+     *
+     */
+    async getPostsByRemoteLookUp(
+        defaultAccount: Account,
+        handle: string,
+        next: string,
+    ): Promise<GetProfileDataResult | Error> {
+        const context = this.fedifyContextFactory.getFedifyContext();
```

Comment:
This function has the exact same code deleted from `/profile.ts`

Reply:
I have not used the `remoteRepository` here in this PR because by the time we discussed that I already did the BE changes. I'll do that refactoring incrementally after shipping these changes.